src/components/vehicleDataGroup/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { BoxProps as ChakraProps } from '@chakra-ui/react';
+
+import { Box, H4, List } from '../';
+
+interface VehicleDataGroupProps extends ChakraProps {
+  title?: string;
+}
+
+const VehicleDataGroup: FC<PropsWithChildren<VehicleDataGroupProps>> = ({
+  title,
+  children,
+  ...rest
+}) => (
+  <Box {...rest}>
+    {title ? <H4>{title}</H4> : null}
```

Comment:
shouldn't title be mandatory? I don't see a usecase where we don't have it

Reply:
also, you probably want to use a normal text and apply heading4 as a text style. Reason is that this component can be used however the user wants to. It could break the correct heading nesting of the page